ta.json.template

```diff
@@ -0,0 +1,174 @@
+{
```

Comment:
please delete this file `ta.json.template` as it is a duplicate of the translation.

Only the `en.json.template` is necessary as it is works as the template for all translations